src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -651,6 +661,30 @@ private void sendReportTrackToEditor(String eventName) {
             }
         }
 
+        private void sendLayoutError(JSONObject errorMessage) {
+            if (mEditorConnection == null || !mEditorConnection.isValid()) {
+                return;
+            }
+
+            final OutputStream out = mEditorConnection.getBufferedOutputStream();
+            final OutputStreamWriter writer = new OutputStreamWriter(out);
+
+            try {
+                errorMessage.put("type", "layout_error");
```

Comment:
The reason I'm doing this is because I want to have this exception for all the layout exceptions. And JSONObject is the best choice since it's flexible on its content. I'm expecting the linear layout update will have some sort of exception.

But, I'll check out the existing error message.